_harp/apply/_apply.md

```diff
@@ -18,66 +18,55 @@
   Complete the [Expression of Interest](http://www.foundersandcoders.com/apply/interest.html) form.
 
 
-  When we receive your Expression of Interest, we will invite you to our Coding for Everyone community on [Gitter](https://gitter.im/codingforeveryone). Please join us and introduce yourself.
+  When you send your Expression of Interest, you DO NOT register to the course, in order to do so please see STEP 3.
```

Comment:
Everywhere else, we say "apply" rather than "register". Also "you do not register/apply" sounds like an order.

Instead, maybe:

> Sending your Expression of Interest is NOT the same as submitting an application to the course.
>
> To submit an application, got to STEP 3.